dependencies.gradle

```
@@ -65,6 +65,12 @@ dependencies {
     modImplementation(forge.javd) { transitive = false }
     modRuntimeOnly("curse.maven:trenzalore-870210:4848244")
 
+    modCompileOnly(forge.ftbteams)
+    modCompileOnly(forge.ftblibrary)
+
+    modImplementation(forge.argonauts)
+    modImplementation(forge.resourceful)
```

These probably shouldn't be hard dependencies

```suggestion
    modCompileOnly(forge.argonauts)
    modCompileOnly(forge.resourceful)
```